src/components/ProgramRecord/ProgramRecordHeader.jsx

```diff
@@ -77,15 +77,18 @@ function ProgramRecordHeader({
         </span>
       </div>
       <div className="learner-info my-3">
-        <FormattedMessage
-          id="learner.username"
-          defaultMessage="{username}"
-          description="Username of the learner"
-          values={{
-            username: learner.username,
-          }}
-        />
-        |
+        {
+          learner.full_name && (
+          <FormattedMessage
+            id="learner.full_name"
+            defaultMessage="{full_name} | "
```

Comment:
@ahtesham-quraish -- Having the `|` character as part of the translated string seems... strange. Also, I question why this needs to be a `FormattedMessage` here when it is the name of a user, which will never be translated.